JavaScript/io.js

```diff
@@ -0,0 +1,75 @@
+const prompt = require("prompt-sync")({ sigint: true });
```

Comment:
It doesn't really matter now, but in the future if the person who wants to use your code needs to install something, try to add it to the README.